### PR TITLE
added multiQC, moved trimming

### PR DESCRIPTION
**Pull request by Gerbenvandervries**

### CODE REVIEW

File: generate_template.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-module load NGS_RNA/3.2.4-Molgenis-Compute-v16.05.1-Java-1.8.0_45
+module load NGS_RNA/3.2.5
 module list
 
 HOST=$(hostname)
```
Comment (RoanKanninga) [inline generate_template.sh]: you can use host -s (for shortname), than you don't have to do the HOST%%.*

File: protocols/CreateInhouseRnaSeqProjects.sh [hunk 2]
```diff
@@ -130,6 +130,7 @@ fi
 echo "before run second rocket"
 echo pwd
 
+module load Molgenis-Compute/v16.05.1-Java-1.8.0_45
```
Comment (RoanKanninga) [inline protocols/CreateInhouseRnaSeqProjects.sh]: Why not loading RNA pipeline instead (now you can get differences between the RNA pipeline compute and loading this hardcoded compute)

File: protocols/MultiQCReport.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            in=0
+            break
+        fi
+    done
+    return $in
+}
+
+cat > ${intermediateDir}/${project}_QC_config.yaml <<'_EOF'
+
+report_header_info:
```
Comment (mmterpstra) [inline protocols/MultiQCReport.sh:56]: Also add costom logo and stuff (in the future).
details here
https://github.com/ewels/MultiQC/blob/master/multiqc_config_example.yaml#L93

Comment (Gerbenvandervries) [reply protocols/MultiQCReport.sh:56]: ah nice!

**Review by RoanKanninga — APPROVED** [review]

**Pull request merged**